ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -437,41 +442,45 @@ try:
                                               '--type')
 
         key_pair_name = args.sshName
+        ssh_private_key_file = args.privateKey
+        if not key_pair_name and not ssh_private_key_file and not \
+                args.accountName:
+            key_pair_name, ssh_private_key_file = \
+                setup.create_upload_key_pair()
```

Comment:
Should fit on the line above

Reply:
Nope as it's `80` characters long. The default line length limit is `79` characters.